template.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: backstage.io/v1alpha1
+kind: Template
+metadata:
+  name: java-client-library
+  title: Java client library template
+  description: Java client library, allows creating client given a swagger link.
+  tags:
+    - recommended
+    - swagger
+    - java-client
+spec:
+  owner: Platform Engineering (#rpe)
+  templater: cookiecutter
+  type: library
+  schema:
+    required:
+      - component_id
+      - swagger_path
+      - base_package
+    properties:
+      component_id:
+        title: Component name
+        type: string
+        description: Name of the component, e.g. idam-client-library. This will be used as artifact id.
```

hmm, we normally include the language in the component name, i guess the repo and component name don't need to match, an alternative would be stripping out java in the template from artifact id, thoughts?
```suggestion
        description: Name of the component, e.g. idam-java-client. This will be used as artifact id.
```